FWCore/Catalog/src/FileLocator.cc

```diff
@@ -220,7 +228,9 @@ namespace edm {
 
     /*first of all do the lfn-to-pfn bit*/
     {
-      DOMNodeList* rules = doc->getElementsByTagName(_toDOMS("lfn-to-pfn"));
+      XMLCh* lfn_to_pfn = XMLString::transcode("lfn-to-pfn");
```

Comment:
The code would have to move somewhere else since `FWCore/Catalog` can't depend on `DetectorDescription/Parser`.